lib/runner/cli/cli.js

```diff
@@ -138,16 +138,22 @@ class CliRunner {
   }
   
   setMobileOptions(argv) {
-    const {desiredCapabilities} = this.test_settings;
+    const {desiredCapabilities, selenium} = this.test_settings;
 
-    if (!isRealIos(desiredCapabilities)) {
-      return;
+    if (isRealIos(desiredCapabilities) && !selenium.use_appium) {
```

Comment:
`selenium` can also be `undefined` here, which will throw an error. Maybe Line 141 can be changed to:
```js
const {desiredCapabilities, selenium = {}} = this.test_settings;
```

Reply:
fixed [83a9e00](https://github.com/nightwatchjs/nightwatch/pull/3572/commits/83a9e00c0c680dd07b9ac8fac6f55835a9bb1acd)